SMBPositionAnalytics.md

```diff
@@ -0,0 +1,151 @@
+
+https://api.careerbuilder.com/core/positionanalytics/carotene
+
+https://api.careerbuilder.com/core/positionanalytics/job
+
+Requests can be sent in the form of HTTP GET or POST (form or JSON)
+
+ This service requires Careerbuilder Oauth 2.0 client credentials. For more information please see our ReadMe [here](/Readme.md#access).
+
+## Requests
+### Carotene With Location Request
+A request to the carotene endpoint provides data on how a specified job title performs in a given location.
+
+E.g. I'm a recruiter and I'm about to post a Software Engineer job in Atlanta. How many applies have there been to Software Engineer titles in this city in the last day? What is the change from the day before? On average how long does it take to fill a Software Engineer role in this location? This endpoint serves to answer these questions.
+
+Requests consist of:
+
+| Param | Type | Description |
+|-------|------|-------------|
+| `carotene_id` | string | Unique carotene identifier v3.1.
+| `city` | string | Full name of a city respective of the carotene title
+| `state` | string | Two letter abbreviation for the state
+
+Example cURL request:
+
+```
+curl -X POST \
+  https://api.careerbuilder.com/core/positionanalytics/carotene \
+  -H 'Accept: application/json;version=1.0' \
+  -H 'Authorization: <BEARER_TOKEN>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"carotene_id": "11.0",
+	"city": "Atlanta",
+	"state": "GA",
+      }'
+```
+
+Currently only Carotene v3.1 is supported. See the documentation for [Job Title service](https://github.com/careerbuilder/DataScienceAPIDocumentation/blob/master/JobTitle.md)
+for more information about carotene classification and carotene IDs.
+
+### Job Request
+A request to the job endpoint requires a CareerBuilder Job Identifier also known as a Job DID and serves data pertaining to active jobs(open positions). Relevant data points returned include: number of clicks, number of applies, number of views, and total number of actions.
+
+Requests consist of:
+
+| Param | Type | Description |
+|-------|------|-------------|
+| `job_did` | string | Unique CareerBuilder job identifier.
+
+
+Example cURL request:
+
+```
+curl -X POST \
+  https://api.careerbuilder.com/core/positionanalytics/job \
+  -H 'Accept: application/json;version=1.0' \
+  -H 'Authorization: <BEARER_TOKEN>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"job_did": "J2N21P76BM530W9HRQG",
+      }'
+```
+
+## Responses
+The data returned in the response will contain an object or group of objects representing actions taken on a carotene title or job respectively. Each action will have associated `current` and `delta` values. The `current` value will account the total number of times an action was performed over the last day; the `delta` will identify the change in percentage of the `current` value from the day before. Positive values in the `delta` identify an increase day over day and negative values identify a decrease.
```

Comment:
'account' is kind of a strange verb to use here imo, maybe 'will be' or 'is'?

Reply:
> the `delta` will identify the change in percentage of the `current` value from the day before.

Is that correct? I thought it was the delta between the current day of the week and the same day of the week 7 days ago?